src/integration/pages/onDemandRadioEpisode/indonesia/__snapshots__/amp.test.js.snap

```diff
@@ -138,13 +138,31 @@ exports[`AMP On Demand Radio Episode SEO Lang attribute 1`] = `"id"`;
 exports[`AMP On Demand Radio Episode SEO Linked data 1`] = `
 Object {
   "@context": "http://schema.org",
-  "@type": "AudioObject",
-  "description": "Berita terbaru dari seluruh dunia dan ulasan peristiwa dari Indonesia. Juga berita olahraga terbaru dan berbeda setiap harinya.",
-  "duration": "PT15M30S",
-  "embedURL": "https://polling.test.bbc.co.uk/ws/av-embeds/media/indonesia/bbc_indonesian_radio/w172xh267fpn19l/id/amp?morph_env=live",
-  "name": "Dunia Pagi Ini",
-  "thumbnailUrl": "https://ichef.bbci.co.uk/images/ic/1024x576/p08b4828.png",
-  "uploadDate": "2020-04-27T00:00:00.000Z",
+  "@type": "RadioChannel",
```

Comment:
This seems incorrect - should it be an `AudioObject` rather than `RadioChannel` based on https://github.com/bbc/simorgh/pull/6745#discussion_r433836019